test/test_common/environment.cc

```diff
@@ -63,7 +63,11 @@ std::string getOrCreateUnixDomainSocketDirectory() {
   // Generate temporary path for Unix Domain Sockets only. This is a workaround
   // for the sun_path limit on sockaddr_un, since TEST_TMPDIR as generated by
   // Bazel may be too long.
+#if !defined(WIN32)
   char test_udsdir[] = "/tmp/envoy_test_uds.XXXXXX";
+#else
+  char test_udsdir[] = "c:\\Windows\\TEMP\\envoy_test_uds.XXXXXX";
+#endif
```

Comment:
Can this be generalized into some type of helper function that gets a temp directory vs. having this logic split here and elsewhere?

Reply:
👍